docs/3.api/1.components/4.nuxt-link.md

```diff
@@ -185,8 +186,13 @@ interface NuxtLinkOptions {
   externalRelAttribute?: string;
   activeClass?: string;
   exactActiveClass?: string;
-  prefetchedClass?: string;
   trailingSlash?: 'append' | 'remove'
+  prefetch?: boolean
+  prefetchedClass?: string
+  prefetchOn?: Partial<{
```

Comment:
```suggestion
  prefetchOn?: 'visibility' | 'interaction' | Partial<{
```

Reply:
This deliberately forces using object syntax with `defineNuxtLink` as it's already a more 'custom' API